angr/procedures/libc/strncmp.py

```diff
@@ -91,6 +91,13 @@ def run(self, a_addr, b_addr, limit, a_len=None, b_len=None): #pylint:disable=ar
                 variables |= a_byte.variables
                 variables |= b_byte.variables
 
+                if ignore_case:
+                    # convert both to lowercase
+                    if ord('a') <= a_conc <= ord('z'):
+                        a_conc -= ord(' ')
+                    if ord('a') <= b_conc <= ord('z'):
+                        a_conc -= ord(' ')
```

Comment:
found a bug! that should be `b_conc -=`.

Reply:
Good catch, thanks!